components/Nav/RightNav.jsx

```diff
@@ -0,0 +1,89 @@
+import Link from 'next/link';
+import styled from 'styled-components';
+
+/*This code is for the style of right navbar which is changed when screen size is mobile. Also the if statement inside style components is for underline transition for the nav options when the screen is not mobile size*/
+const Ul = styled.ul`
+  list-style: none;
+  display: flex;
+  flex-flow: row nowrap;
+  font-size: 0.5em;
+  li {    
+    padding-right: 3em;
+    padding-top: 1em;
+    cursor: pointer;    
+  }    
+  ${ function ({open}){
+    if (open==false) {
+      return (
+        `.hover-underline-animation {
+          display: block;
+          position: relative;    
+        }
+        
+        .hover-underline-animation:after {
```

Comment:
- Make sure the animation is consistent with the text

Reply:
it should have worked with display: block; in line 19 , but it didnt any alternative suggestion?

Reply:
Try centering the text in the middle and see if it looks good